scoap3/articles/api/views.py

```diff
@@ -54,10 +58,24 @@ class ArticleDocumentView(DocumentViewSet):
     serializer_class = ArticleDocumentSerializer
 
     filter_backends = [SearchFilterBackend]
+    renderer_classes = api_settings.DEFAULT_RENDERER_CLASSES + [ArticleCSVRenderer]
 
     search_fields = ("title", "id")
     permission_classes = [IsAuthenticatedOrReadOnly]
 
+    def get_renderer_context(self):
+        renderer_context = super().get_renderer_context()
+        requested_renderer_format = self.request.accepted_media_type
+        if "text/csv" in requested_renderer_format:
+            pagination.PageNumberPagination.page_size = 100
+        return renderer_context
+
+    def get_serializer_class(self):
+        requested_renderer_format = self.request.accepted_media_type
+        if "text/csv" in requested_renderer_format:
+            return SearchCSVSerializer
+        return ArticleDocumentSerializer
```

Comment:
do we need to hardcode the serialisers in the viewset? Did we check of a config var or something similar?